fastembed/text/pooled_embedding.py

```diff
@@ -101,7 +103,11 @@ def list_supported_models(cls) -> list[dict[str, Any]]:
         Returns:
             list[dict[str, Any]]: A list of dictionaries containing the model information.
         """
-        return supported_pooled_models
+        return cls.supported_models
+
+    @classmethod
+    def add_custom_model(cls, model_info: dict[str, Any]):
+        cls.supported_models.append(model_info)
```

Comment:
there is not need to redefine this method in the subclasses if they fetch the right cls variable